markdowns/workingGroup/meetings.md

```diff
@@ -352,6 +352,119 @@ Slides available at: https://docs.google.com/presentation/d/1mMPIVfjFBOh9qHL5aW4
 
 **10th December 2020**
 
+
+SUMMARY:
+<ol type = "1"> 
```

Comment:
Requesting the HTML  tags to be removed and be replaced with the text in markdown format. The rest of the doc has been migrated to the same, can be used as a reference.